1.9/api/index.md

```diff
@@ -0,0 +1,38 @@
+---
+post_title: DC/OS API Reference
+nav_title: API Reference
+menu_order: 4
+---
+
+The DC/OS API is a collection of routes backed by [DC/OS Components](/docs/1.9/overview/architecture/components/) made available through [Admin Router](/docs/1.9/overview/architecture/components/#admin-router), an API gateway.
+
+To determine the address of your cluster, see [Cluster Access](/docs/1.9/api/access/).
+
+
+## Routing
+
+**Admin Router Master** runs on each master node and serves as the primary API gateway for interaction with DC/OS components.
+
+See [Master Routes](/docs/1.9/api/master-routes/) for a list of routes available on master nodes.
+
+**Admin Router Agent** runs on each agent node and provides routes for monitoring, debugging, and administration.
+
+Some agent routes, like logs and metrics, are proxied through the master Admin Router to allow external access.
+Other routes, like component management, are for internal use only.
+
+See [Agent Routes](/docs/1.9/api/agent-routes/) for a list of routes available on agent nodes.
+
+![DC/OS API Routing](/docs/1.9/api/img/dcos-api-routing.png)
+
+
+## Route Types
+
+Individual routes are generally one of three types:
+
+**Proxy** routes retrieve resources from another address.
```

Comment:
If I do a list, should I also make the above section a list with the `Admin Router` and `Admin Router Agent` explanations? It's using a similar style.